plugins/link/plugin.js

```diff
@@ -895,7 +895,16 @@
 		 * @cfg {Boolean} [linkShowTargetTab=true]
 		 * @member CKEDITOR.config
 		 */
-		linkShowTargetTab: true
+		linkShowTargetTab: true,
+
+		/**
+		 * Default URL protocol used for link dialog.
+		 *
+		 * @cfg {String}
+		 * @member CKEDITOR.config
+		 * @since 4.13.0
+		 */
+		link_defaultProtocol: 'http://',
```

Comment:
Shouldn't we be consistent with rest of the config options and have `linkDefaultProtocol`?

Reply:
`link_defaultProtocol` is actually consistent with our current guideline for config values - but as you noticed other link config values have full camel case names, so I agree it's probably better to stick with old naming if it's already available :+1: